src/tiles/item-image.ts

```diff
@@ -111,12 +113,14 @@ export class ItemImage extends LitElement {
   }
 
   private get itemImageClass(): ClassInfo {
-    const toBlur = this.model?.contentWarning || this.model?.loginRequired;
+    const toBlur =
+      !this.suppressBlurring &&
+      !!(this.model?.contentWarning || this.model?.loginRequired);
```

Comment:
Minor readability suggestion: There's a lot going on in this conditional. I might break this up into intermediate variables to make it more obvious what's happening.